spec/services/fetch_job_creator_spec.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe FetchJobCreator do
+  let(:collection) do
+    create(:collection, fetch_start_month: '2018-10-01')
+  end
+
+  before do
+    FetchMonth.destroy_all
+    allow(Date).to receive(:today).and_return(Date.parse('2019-08-21'))
+    # TODO: allow(FetchMonthJob).to receive(:enqueue_later)
+  end
+
+  context 'for a collection that has fetch months' do
+    before do
+      collection.fetch_months.create!(year: 2018, month: 11, status: 'waiting')
+    end
+
+    it 'creates fetch_months' do
+      described_class.run(druid: collection.druid)
+      expect(collection.fetch_months.map { |m| [m.year, m.month] }).to eq [
+        [2018, 11], [2018, 12], [2019, 1], [2019, 2]
+      ]
+      # TODO: expect(FetchMonthJob).to have_received(:enqueue_later).exactly(4).times
+    end
+  end
```

Comment:
According to @nullhandle, one common case will be a 0 month embargo. I think a test of this would be a good idea.